Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:6.10.2-alpine
+FROM node:10.15-alpine
```

The node version upgrade is out of scope for this PR
```suggestion
FROM node:10.14.2-alpine
```